src/chains/optimism/eips.ts

```diff
@@ -0,0 +1,48 @@
+import { EIP, EIPState } from '@/types/eip';
+import { eip1559 as eip1559OnEthereum, eips as ethereumEIPs } from '../mainnet/eips';
+import { MainnetHardfork, getHardforksFrom } from '../mainnet/hardforks';
+
+const eip1559OnOptimism: EIP = {
+  ...eip1559OnEthereum,
+  parameters: [
+    {
+      name: 'INITIAL_BASE_FEE',
+      value: 1000000000,
+    },
+    {
+      name: 'BASE_FEE_MAX_CHANGE_DENOMINATOR',
+      value: 250,
+    },
+    {
+      name: 'ELASTICITY_MULTIPLIER',
+      value: 6,
+    },
+  ],
+  references: [
+    ...eip1559OnEthereum.references,
+    'https://docs.optimism.io/chain/differences#eip-1559-parameters',
+  ],
+};
+
+const hardforksFromShanghai: string[] = getHardforksFrom(MainnetHardfork.Shanghai);
+const eip6049: EIP = {
```

Comment:
This EIP is actually present on mainnet too even though its not listed in the execution specs (IMO it should be listed there). The reason for it is that it's just a meta-EIP that makes no changes but conveys changes will come. You can see https://blog.ethereum.org/2023/03/28/shapella-mainnet-announcement as the source for this